src/views/Home/components/Schedule/components/index.scss

```diff
@@ -0,0 +1,118 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.schedule-table {
+  position: relative;
+  width: 100%;
+
+  &__table {
+    min-height: 300px;
+    margin: 0 50px;
+    padding: 20px 30px 40px;
+    background: linear-gradient(180deg, rgba(91, 62, 128, 0.64) 0%, rgba(86, 114, 170, 0.78) 100%);
+    box-shadow: 0 4px 8px rgba(193, 194, 207, 0.74);
+    border-radius: 20px;
+
+    font-family: "Lato", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+
+    @include responsiveness.screen(smallest, phone) {
+      margin: 0;
+      padding-left: 10px;
+      padding-right: 10px;
+    }
+  }
+
+  &__header {
+    padding: 0 0 10px;
+    text-align: center;
+    border-bottom: 2px solid #9fc5ff;
+
+    font-weight: bold;
+    color: #ced6fa;
+    font-size: 28px;
+
+    @include responsiveness.screen(smallest, phone) {
+      font-size: 20px;
+    }
+  }
+
+  &__event {
+    display: flex;
+    flex-wrap: nowrap;
+    border-bottom: 2px solid #9fc5ff;
+    color: #d3ddff;
+    font-size: 20px;
+    width: 100%;
+
+    @include responsiveness.screen(smallest, phone, tablet) {
+      font-size: 16px;
+    }
+
+    &__time {
+      margin: 0;
```

Comment:
On desktop, I think the margin should be a little bit higher according to the figma